### PR TITLE
AT_ 01.10.04 | Registering is not possible with any letters in 'Phone number' input

### PR DESCRIPTION
**Pull request by SvetlanaKanel**
https://trello.com/c/6XkA73Xz/835-at011004-login-by-phone-negative-registering-is-not-possible-with-any-letters-phone-number-input

### CODE REVIEW

File: cypress/pageObjects/StartPage.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ export class LoginPopup {
     getPhoneNumberInputLabel = () => cy.get('div[class="col-sm-6 col-xs-8"] label');
     getLoginPopupModal =() => cy.get('#loginModal .modal-content');
     getEmailErrorMessage = () => cy.get('.alert.alert-danger');
+    getPhoneNumberAlert = () => cy.get('.alert-danger');
```
Comment (elviraKh1) [inline cypress/pageObjects/StartPage.js:65]: Look like 64 and 65 lines are duplicates - is it a general alert?

File: cypress/e2e/testUiAndFunction/startPage/US_01.10_LoginByPhoneNegative.cy.js [hunk 2]
```diff
@@ -44,4 +44,14 @@ describe('US_01.10 | Login by phone negative', () => {
             .should('be.visible')
             .and('have.text', this.startPage.alert.loginPopupByPhoneMessageAlert);
     })
+
+    it('AT_01.10.04 | Registering is not possible with any letters in "Phone number" input', function() {
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/US_01.10_LoginByPhoneNegative.cy.js:48]: It's not registration, it's login.

File: cypress/pageObjects/StartPage.js (hunk 1 above)
Comment (SvetlanaKanel) [reply cypress/pageObjects/StartPage.js:65]: Maybe, but it is different elements on different popup windows

Comment (Rom-T) [reply cypress/pageObjects/StartPage.js:65]: Better leave the one on line 52, as it makes sense for each popup tab

File: cypress/e2e/testUiAndFunction/startPage/US_01.10_LoginByPhoneNegative.cy.js (hunk 2 above)
Comment (SvetlanaKanel) [reply cypress/e2e/testUiAndFunction/startPage/US_01.10_LoginByPhoneNegative.cy.js:48]: you are right!

**Review by Rom-T — APPROVED** [review]

**Pull request merged**